engine/src/main/java/org/destinationsol/game/input/UiControlledPilot.java

```diff
@@ -75,6 +76,11 @@ public Faction getFaction() {
         return Faction.LAANI;
     }
 
+    @Override
+    public void setFaction(String faction) {
+    // TODO
```

Comment:
Helps if you leave some additional notes in a TODO (especially because some tools will extract the TODO meaning you lose the context and just see "TODO")